.github/workflows/release.yml

```diff
@@ -0,0 +1,42 @@
+name: Create Release
+on:
+  repository_dispatch:
+    types: az_bootstrap_release
+
+jobs:
+  release:
+    name: Create Release
+    runs-on: ubuntu-18.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ssh-key: ${{ secrets.SELF_DEPLOY_KEY }}
+
+      - name: Get new updates from arizona-bootstrap
+        run: |
+          distwhitelist='css js'
+          srcroot='dist'
+          clonedir='/tmp/arizona-bootstrap'
+          git config --global user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+          git config --global user.name "${GITHUB_ACTOR}"
+          git clone git@github.com:az-digital/arizona-bootstrap.git "${clonedir}"
+          (cd "${clonedir}" && git checkout ${{ github.event.client_payload.ref }})
+          for w in $distwhitelist ; do
+            rsync --recursive --delete "${clonedir}/${srcroot}/${w}" "${w}"
+          done
+          git add .
+          git commit -m "Arizona Bootstrap updates for ${{ github.event.client_payload.version }}"
+          git push
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: "v${{ github.event.client_payload.version }}"
```

Comment:
We might need to look into whether the NPM-style version tags (e.g. `v2.0.0`) work for composer/packagist or not.  It might be necessary to omit the `v` so that projects wanting to use this composer package as a dependency can use composer's version constraints as expected...

Reply:
Actually, it doesn't look like the `v`'s are present in Twitter's packagist repo for Bootstrap:
https://packagist.org/packages/twbs/bootstrap

Reply:
Never mind.  Composer's docs explicitly state that a `v` prefix in a tag name is OK 🎉 :
>Note that Composer automatically removes the 'v' prefix in the actual tagname to get a valid final version number.

https://getcomposer.org/doc/articles/versions.md#tags